lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -128,6 +136,27 @@ def self.download_file_from_tag(repository:, tag:, file_path:, download_folder:)
 
         download_path
       end
+
+      # Creates (or updates an existing) GitHub PR Comment
+      def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier:)
+        comments = github_client.issue_comments(project_slug, pr_number)
+
+        existing_comment = comments.detect do |comment|
+          comment.body.include?(reuse_identifier)
```

Comment:
This seems like a pretty fragile detection. Like if I use "installable" as `reuse_identifier` and it happens that someone commented with "Hey <someone> can you please test the installable build once ready" then it will pick up that comment instead of the expected one.

I'd instead suggest to detect the whole `<!-- REUSE_ID: #{reuse_identifier} -->` string, and also, as a bonus security measure, only search amongst comments made by the bot and not comments from other, human users.